MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,221 @@
+
+import PureLayout
+import UIKit
+import MovieAppData
+
+class MovieDetailsViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate {
+    
+    private var movieImageView: UIImageView!
+    private var starIconImageView: UIImageView!
+    private var ratingLabel: UILabel!
+    private var userScoreLabel: UILabel!
+    private var titleLabel: UILabel!
+    private var releaseYearLabel: UILabel!
+    private var dateLabel: UILabel!
+    private var categoriesLabel: UILabel!
+    private var durationLabel: UILabel!
+    private var overViewLabel: UILabel!
+    private var descriptionLabel: UILabel!
+    private var collectionView: UICollectionView!
```

Comment:
I know you predicted the use of stack views here, but i decided to try out collection view on this simple example to learn more and i would like to leave it because i was so happy that i made it work :P.